### DISCUSSION TITLE
Problem installing the project on my system

### DISCUSSION THREAD

**Discussion by TechGuySaad** [khattakdev/minimalcommit#9] [Q&A]
I have cloned the project on my system and tried to use the npm install command provide by the maintainer, but I ham having below error
![Screenshot from 2023-04-04 19-51-47](https://user-images.githubusercontent.com/81616266/229832554-2221b56c-f5de-483b-8f23-368d6161f114.png)

**Comment by khattakdev** [comment]
@TechGuySaad add the prefix `sudo` and it should work. Currently, the `npm` does not have access to install anything globally, so you need to run it as administrator.

**Reply by TechGuySaad** [reply]
All good! Works Now! Thanks!

**Comment by TechGuySaad** [comment]
problem is solved with sudo
